vyxal/elements.py

```diff
@@ -2835,8 +2842,10 @@ def repeat(lhs, rhs, ctx):
         @lazylist
         def gen():
             prev = value
+            val = value
```

Comment:
Why are there variables called `val` and `value`? Could they be called something more descriptive please?

Reply:
This has been clarified. Thank you for noticing this. :)